bubblewrap.c

```diff
@@ -1882,6 +1892,57 @@ parse_args_recurse (int          *argcp,
 
           opt_seccomp_fd = the_fd;
 
+          argv += 1;
+          argc -= 1;
+        }
+      else if (strcmp (arg, "--userns") == 0)
```

Comment:
should we change the condition at line 2342 and accept `--uid` when `--userns` is specified?  e.g.

currently it fails like:
```
$ ./bwrap --uid 0 --bind / / --userns 3 true
bwrap: Specifying --uid requires --unshare-user
```

Reply:
When we're reusing a namespace we assume it was set up already, including user mapping, so we don't need (nor want) to change the uid mappings. Do we use the uid passed in for anything else? I don't think so?

Reply:
I was testing it with a user namespace that was configured separately and with multiple IDs available (with newuidmap/newgidmap).

If that is out of scope, then LGTM